src/main/java/com/keboola/tableexporter/MetaFetcher.java

```diff
@@ -13,6 +13,8 @@
 
     private Connection connection;
 
+    private final String systemOwners = "('SYS', 'SYSTEM', 'MDSYS', 'DMSYS', 'CTXSYS', 'WMSYS', 'XDB', 'APEX_040000')";
```

Comment:
CSAS tam ma aj `WMSYS` systemoveho ownera, ktoreho v nasej testovacej Oracle nemame.
Rovno som to dal ako konstantu.